Example/Backpack/View Controllers/LabelsPerformanceViewController.swift

```diff
@@ -0,0 +1,62 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import Backpack
+
+class LabelsPerformanceViewController: UIViewController {
+    @IBOutlet weak var verticalStackView: UIStackView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        for i in 0..<100 {
+            let emphasized = i % 2 == 0
+            let view = LabelsPerformanceViewController.buildHorizontalStackView(emphasized: emphasized)
+            verticalStackView.addArrangedSubview(view)
+        }
+    }
+
+    static func buildHorizontalStackView(emphasized: Bool) -> UIStackView {
+        let horizontalStackView = UIStackView(frame: CGRect.zero)
+        horizontalStackView.axis = .horizontal
+        horizontalStackView.spacing = BPKSpacingSm
+        horizontalStackView.distribution = .equalSpacing
+
+        let l1 = Backpack.Label(fontStyle: emphasized ? BPKFontStyle.textXs : BPKFontStyle.textXsEmphasized)
```

Comment:
Could we iterate over a list of text-styles instead to generate these without so much duplication?

Reply:
Yeah sure

Reply:
Done